regression/ansi-c/always_inline1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+^CONVERSION ERROR$
```

Comment:
Add a comment to the test specifying *why* we expect an error

Reply:
Done.